youtube_dl/extractor/rtvslo.py

```diff
@@ -0,0 +1,61 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import (
+    unified_timestamp,
+    try_get, ExtractorError
+)
+
+
+class RTVSLO4DIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:4d|www)\.rtvslo\.si/(?:arhiv/.+|embed|4d/arhiv)/(?P<id>\d+)'
+    _TEST = {
+        'url': 'https://4d.rtvslo.si/arhiv/seje-odbora-za-kmetijstvo-gozdarstvo-in-prehrano/174595438',
+        'md5': 'b87e5a65be2365f83eb0d24d44131d0f',
+        'info_dict': {
+            'id': '174595438',
+            'ext': 'mp4',
+            'title': 'Krajčič o tatvini sendviča',
+            'thumbnail': r're:https://img.rtvslo.si/.+\.jpg',
+            'timestamp': 1549999614,
+            'upload_date': '20190212',
+            'duration': 85
+        },
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        embed_url = 'https://4d.rtvslo.si/embed/' + video_id
+        embed_html = self._download_webpage(embed_url, video_id)
+
+        client_id = self._search_regex(r'\[\'client\'\] = "(.+?)";', embed_html, 'clientId')
+
+        info_url = 'https://api.rtvslo.si/ava/getRecordingDrm/' + video_id + '?client_id=' + client_id
```

Comment:
use `getRecording` and remove `getMedia` call.

Reply:
The official apps don't use that call, but it does seem to exist and work. I'll replace it